### DISCUSSION TITLE
Azure Data Studio - Restore Database

### DISCUSSION THREAD

**Discussion by rafaferreirasantos** [balta-io/2811#16] [Q&A] [answered]
Senhores, a opção Data-tier Application wizard não aparece no meu Azure Data Studio. Fiz a instalação do SSMS 18.11.1 (with Azure Data Studio) mas mesmo assim as opções que aparecem quando clico com o botão direito no nome do server são bem restritas.

Alguém sabe o que posso estar fazendo de diferente?

![Screenshot_3](https://user-images.githubusercontent.com/7022211/159175262-663969f2-37b3-4e47-b97c-3e2b074af8f2.png)
![Screenshot_1](https://user-images.githubusercontent.com/7022211/159175272-fbca4ddd-973b-48d9-b9d6-d97c4f806e90.png)

**Comment by BrewertonSantos** [comment, marked as answer]
Olá @zecaloteiro durante o curso o 2805 (Fundamentos do SQL Server) o Balta mostra que para ter este recurso é necessário instalar o pacote _Admin pack for SQL Server_ no azure data studio. Após isto você terá esta opção habilitada.

![image](https://user-images.githubusercontent.com/55927647/159177593-4638cfcf-a581-44fc-a520-97019454f0f1.png)
